fe/src/main/java/org/apache/doris/qe/ShowExecutor.java

```diff
@@ -1053,8 +1054,8 @@ private void handleShowDelete() throws AnalysisException {
         }
         long dbId = db.getId();
 
-        Load load = catalog.getLoadInstance();
-        List<List<Comparable>> deleteInfos = load.getDeleteInfosByDb(dbId, true);
+        DeleteHandler deleteHandler = catalog.getDeleteHandler();
```

Comment:
You should also show the delete info from `Load`, or after we upgrade the Doris, the old delete info can not be seen.

Reply:
done